web/src/engine/keyboard/src/defaultRules.ts

```diff
@@ -131,6 +131,13 @@ export default class DefaultRules {
 
   // Should not be used for mnenomic keyboards.  forAny()'s use of this method checks first.
   public forNumpadKeys(Lkc: KeyEvent) {
+    // Do not provide output if modifiers other than Shift are held.
+    // There may be associated shortcuts we might otherwise block.
+    // See #12483 - zooming shortcuts can trigger from CTRL & + or - on the numpad
+    if(Lkc.Lmodifiers != ModifierKeyConstants.K_SHIFTFLAG && Lkc.Lmodifiers != 0) {
+      return null;
+    }
```

Comment:
Compare/contrast with this code block:

https://github.com/keymanapp/keyman/blob/2bd183c1c01045e9aca2bdc8eaf33aa46e40f788/web/src/engine/keyboard/src/defaultRules.ts#L195-L204

This is what prevents key shortcuts from being blocked for standard keys; it was simply missing for numpad keys.